docs/api/quickstart.md

```diff
@@ -0,0 +1,139 @@
+---
+title: API Quick Start Guide
+---
+
+The Rancher API is a programatic interface for your upstream and downstream clusters. This guide will help get you started on using the API from your client of choice.
+
+1. In the upper left corner, click **☰ > Global Settings**. 
+2. Find and copy the address in the `server-url` field.
+3. [Create](../reference-guides/user-settings/api-keys#creating-an-api-key) a Rancher API key with no scope.
+
+  :::danger
+
+  A Rancher API key with no scope grants unrestricted access to all resources that the user can access. To prevent unauthorized use, this key should be stored securely and rotated frequently.
+
+  :::
+
+4. Create a `kubeconfig.yaml` file. Replace `$SERVER_URL` with the server url and `$API_KEY` with your Rancher API key:
+
+    ```yaml
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - name: "rancher"
+      cluster:
+        server: "$SERVER_URL"
+        # Uncomment the following line if your server is using self-signed certs. This option is not recommended for production
+        # insecure-skip-tls-verify: true
+
+    users:
+    - name: "rancher"
+      user:
+        token: "$API_KEY"
+
+    contexts:
+    - name: "rancher"
+      context:
+        user: "rancher"
+        cluster: "rancher"
+
+    current-context: "rancher"
+    ```
+
+You can use this file with any compatible tool, such as kubectl or [client-go](https://github.com/kubernetes/client-go). For a quick demo, see the [kubectl example](#api-kubectl-example). 
+
+For more information on handling more complex certificate setups, see [Specifying CA Certs](#specifying-ca-certs).
+
+For more information on available kubeconfig options, see the [upstream documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+## API kubectl Example
+
+1. Set your KUBECONFIG environment variable to the kubeconfig file you just created:
+
+  ```bash
+  export KUBECONFIG=$(pwd)/kubeconfig.yaml
+  ```
+
+  :::tip
+
+  Use `kubectl explain` to view the available fields for projects, or complex sub-fields of resources:
```

Comment:
It might be good to add a note here that not all resources are going to have detailed output - only the ones that have been migrated to the public api so far.